environments/prod/main.tf

```diff
@@ -0,0 +1,21 @@
+module "hosting_zone" {
+  source = "../../modules/hosting_zone"
+
+  name = var.base_domain
+}
+
+module "cdn_certificate" {
+  source = "../../modules/certificate"
+
+  domain_name = var.base_domain
+  zone_id     = module.hosting_zone.zone_id
+  providers = { # CF Certificate need to be created in US-EAST-1
+    aws = aws.us-east-1
+  }
+}
+
+module "remote-state-s3-backend" {
```

Comment:
How do you pass the provider for the replica?

Reply:
I've used default value: https://github.com/copotrzebne-pl/infrastructure/blob/new-infra/modules/remote-state-s3-backend/variables.tf#L16

Will be more cleaner to define that implicit here?